src/scss/packages/_utilities-required.scss

```diff
@@ -0,0 +1,3 @@
+@import '../uswds/utilities/utility-fonts';
```

Comment:
I could also see an argument for just putting this all into `_required.scss`.

Reply:
since this file has required in its name...seems like a good fit to me

Reply:
I guess a main counter-point would be that utilities are a prime candidate for something that someone would opt not to include if they're cherry-picking packages. But on the other hand, this file doesn't actually "render" anything on its own in the sense that it would add styles to an otherwise-small CSS bundle... it just makes those utility mixins/functions available.

I think I'll move to `_required.scss`, since it makes things a fair bit simpler, and we also don't have to worry about how to document this or describe it in the changelog.

Reply:
Updated in 12ac3da